### PR TITLE
Changelog update for version 1.3

### PR DESCRIPTION
**Pull request by mshariq-nerd**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -10,13 +10,16 @@
 
 * Renamed `deploy.sh` and `build.sh` to `deploy` and `build`
 * Upgraded to Angular v6 (including compatible TypeScript / CLI versions)
+* Removed skip login functionality. No login required when authentication is optional
```
Comment (monika-Dianomic) [inline changelog.md]: Pick location based host on app load.

File: changelog.md [hunk 2]
```diff
@@ -10,13 +10,16 @@
...
 
 #### Removed
 
 * Input mask external lib
 
 #### Bug Fixes
 * [FIXED] Support bundles download when authentication is mandatory
+* [FIXED] Individual graph data binding on refresh
```
Comment (monika-Dianomic) [inline changelog.md]: Download Support Bundle issue is also fixed in this version.

File: changelog.md [hunk 3]
```diff
@@ -5,18 +5,29 @@
 * e2e Tests HTML Report
 * Add new service with specified plugin
 * `./make_deb` script to create debian package (in `packages/Debian/build)
+* In auth mandatory mode FogLAMP running status on top will show the lock icon button until not logged into system
```
Comment (monika-Dianomic) [inline changelog.md]: In auth mandatory mode, if allowPing is false, FogLAMP running status on top will show the lock icon button until not logged into system

File: changelog.md [hunk 4]
```diff
@@ -5,18 +5,29 @@
...
 
 * Input mask external lib
 
 #### Bug Fixes
-* [FIXED] Support bundles download when authentication is mandatory
+* Support bundles download when authentication is mandatory
+* Fixed syslog display order 
+* and squashed many more :]
 
 #### Known Issues
```
Comment (monika-Dianomic) [inline changelog.md]: Known Issues is blank, it should be removed.

File: changelog.md [hunk 5]
```diff
@@ -52,6 +60,7 @@
...
 
 * Support bundles download via browser is not possible when authentication is mandatory (Please use curl request manually with authorization token header)
+* Logout active session not working for non admin user.
```
Comment (praveen-garg) [inline changelog.md]: should be in 1.3.0

File: README.md [hunk 6]
```diff
@@ -54,85 +54,17 @@ See details [here](windows-nginx-deployment-guide.md)
...
-$ sudo apt remove foglamp-gui
-```
-
-> you may want to check debian package contents with `sudo dpkg -c foglamp-gui-1.3.0.deb` or can install in dev env with `sudo dpkg -i foglamp-gui-1.3.0.deb`
+## Creating and Installing Debian Package
+[Debian packaging and usage](debian-readme.md)
 
 ## Running using Docker 
-[instructions](docker-readme.md)
+[Docker instllation and using docker-compose](docker-readme.md)
```
Comment (monika-Dianomic) [inline README.md]: instllation -> installation

**Pull request merged**